Gemfile

```diff
@@ -76,5 +76,8 @@ gem "inum"
 gem "jquery-rails"
 gem "jquery-ui-rails"
 gem "mini_magick"
+gem "redcarpet"
 gem "ruby-mp3info"
 gem "seed-fu"
+gem 'bootstrap-tooltip-rails'
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [SideCI](https://sideci.com) -->